docs/custody.md

```diff
@@ -0,0 +1,18 @@
+# Custody
+
+### Password
+
+When setting up for the first time, you'll need to set a password.
+
+This password is hashed via `PBKDF2`, see [hashPassword() function](../packages/crypto/src/encryption.ts). It utilizes a pseudorandom function along with a salt value and iteration. The use of a salt provides protection against pre-computed attacks such as rainbow tables, and the iteration count slows down brute-force attacks.
+
+This hashed key+salt is stored in `chrome.storage.session` used for deriving spending keys for wallets later.
```

Comment:
Given a possible scenario of multiple wallets, it may be easier to keep storing the password in session storage to derive the spend key on the fly. If not, we can store spend keys in session storage instead. However, at the moment, we don't have account group ids setup yet, so it's hard to associate spend keys with wallets.